src/settings/Settings.ts

```diff
@@ -598,4 +598,9 @@ export const SETTINGS: {[setting: string]: ISetting} = {
         displayName: _td("Enable experimental, compact IRC style layout"),
         default: false,
     },
+    "dontShowChatEffects": {
```

Comment:
Setting names should always be positive without exception. The double negatives in code make it hard to follow along.

Reply:
Hi @turt2live, this was done based on Nad's suggestion on the linked issue: https://github.com/vector-im/element-web/issues/14676#issuecomment-668631587
should i change it still?

Reply:
I'm going to say yes, please change it. If we really want an inverted setting like this then some rework can be done in the settings store, but that would be out of scope for this PR.